### DISCUSSION TITLE
What is the hash for deployments in artifact_location/hash/artifacts/

### DISCUSSION THREAD

**Discussion by ptabandzelic** [databrickslabs/dbx#491] [Q&A]
How is this hash generated, is there a way to override it, for example, im using logging. FileHandler, and i need to get location of the project automatically in order to store logger on dbfs. How to get this hash?

**Comment by renardeinside** [comment]
Please read more on the artifact storage here - https://dbx.readthedocs.io/en/latest/concepts/artifact_storage/
